src/router/handlers/admin.js

```diff
@@ -0,0 +1,356 @@
+/**
+ * @swagger
+ * /org/:
+ *  post:
+ *    description: Use to create an organization
+ *    parameters:
+ *      - name: ha-api-token
+ *        in: header
+ *        required: true
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *      - name: organization
+ *        in: body
+ *        required: true
+ *        schema:
+ *          type: object
+ *    responses:
+ *      '201':
+ *        description: 'Success'
+ *      '400':
+ *        description: 'Invalid syntax'
+ *      '401':
+ *        description: 'JWT not found in header'
+ *      '403':
+ *        description: 'JWT does not have admin privileges'
+ */
+let createOrganization = async (req, res) => {
+
+};
+
+/**
+ * @swagger
+ * /org/:
+ *  put:
+ *    description: Use to update an organization
+ *    parameters:
+ *      - name: ha-api-token
+ *        in: header
+ *        required: true
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *      - name: organization
+ *        in: body
+ *        required: true
+ *        schema:
+ *          type: object
+ *    responses:
+ *      '200':
+ *        description: Success
+ *      '400':
+ *        description: 'Invalid syntax'
+ *      '401':
+ *        description: 'JWT not found in header'
+ *      '403':
+ *        description: 'JWT does not have admin privileges'
+ *      '404':
+ *        description: 'Not found'
+ */
+let updateOrganization = async (req, res) => {
+
+};
+
+/**
+ * @swagger
+ * /hacks/:
+ *  post:
+ *    description: Use to create a hackathon
+ *    parameters:
+ *      - name: ha-api-token
+ *        in: header
+ *        required: true
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *      - name: hackathon
+ *        in: body
+ *        required: true
+ *        schema:
+ *          type: object
+ *    responses:
+ *      '201':
+ *        description: 'Success'
```

Comment:
It should also probably return an object with the hackathon id (in another file you use the `schema` to define what is returned, and I really like that format!)

Reply:
Can do!